### PR TITLE
AlexPromoTeam

### PR DESCRIPTION
**Pull request by GlobalSG**

### CODE REVIEW

**Comment by GlobalSG** [comment]
https://kusama.subscan.io/extrinsic/18443290-4

**Comment by GlobalSG** [comment]
https://polkadot.subscan.io/extrinsic/16054173-2

File: registry/polkadot.json [hunk 1]
```diff
@@ -1,4 +1,12 @@
 [
+     {
+        "address": "GqPDeJBexSsvEq6gBzrnMCn7KNYbzn9bX9ePCjEDN3EikoV",
```
Comment (stepanLav) [inline registry/polkadot.json:3]: Looks like address is mixed up.
```suggestion
        "address": "15G4hfDNtNhRc82As8Ep2YfvpM5xVdX7De3P9qSdHerGA6wC",
```

File: registry/polkadot.json [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
+        "name": "Alex PromoTeam",
+        "image":  "https://raw.githubusercontent.com/GlobalSG/opengov-delegate-registry/0c1a101956dbd5c5ad30e1997d6974cf2e966b62/images/IMG_5814%204.JPG",
```
Comment (stepanLav) [inline registry/polkadot.json:5]: image link should be:
```suggestion
        "image":  "https://raw.githubusercontent.com/nova-wallet/opengov-delegate-registry/master/images/IMG_5814%204.JPG",
```

File: registry/kusama.json [hunk 3]
```diff
@@ -1,4 +1,12 @@
 [
+     {
+        "address": "GqPDeJBexSsvEq6gBzrnMCn7KNYbzn9bX9ePCjEDN3EikoV",
+        "name": "Alex PromoTeam",
+        "image":  "https://raw.githubusercontent.com/GlobalSG/opengov-delegate-registry/0c1a101956dbd5c5ad30e1997d6974cf2e966b62/images/IMG_5814%204.JPG",
```
Comment (stepanLav) [inline registry/kusama.json:5]: link must lead to our repo:
```suggestion
        "image":  "https://raw.githubusercontent.com/novasamatech/opengov-delegate-registry/master/images/IMG_5814%204.JPG",
```

**Comment by stepanLav** [comment]
Hello @GlobalSG, Thank you for request!
Please fix comments and we will be able merge your changes. This is how it will look in the app:
<details>
  <summary>Spoiler!</summary>

Screenshot 2023-06-23 at 07 35 39
Screenshot 2023-06-23 at 07 35 42
Screenshot 2023-06-23 at 07 35 45

</details>

**Comment by GlobalSG** [comment]
> Hello @GlobalSG, Thank you for request! Please fix comments and we will be able merge your changes. This is how it will look in the app:
> 
> Spoiler!

Thanks for corrections, fixed

**Review by stepanLav — APPROVED** [review]

**Pull request merged**